community/kernel/src/test/java/org/neo4j/kernel/impl/storemigration/MigrationTestUtils.java

```diff
@@ -148,17 +148,24 @@ public static void prepareSampleLegacyDatabase( String version, FileSystemAbstra
 
     public static File findFormatStoreDirectoryForVersion( String version, File targetDir ) throws IOException
     {
-        switch ( version )
+        if ( version.equals( StandardV2_3.STORE_VERSION ) )
```

Comment:
`switch` can't be used here any more because `StandardV2_3.STORE_VERSION` is initialized like this: `String STORE_VERSION = StoreVersion.STANDARD_V2_3.string();` and this is no longer a constant expression, as required by `switch-case`